scripts/release

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
If this needs to be executed in bash it should be `#!/usr/bin/env bash`. `#!/bin/sh` refers to "current system shell" which could be something other than bash.

Reply:
It can be executed in any shell it's not specific to bash I think, executed with `bash ./scripts/release` on package.json because I was having some errors but weren't related, will remove that and preserve the  #!/bin/sh.